tools/wptrunner/wptrunner/environment.py

```diff
@@ -232,22 +230,25 @@ def get_routes(self):
                  "text/javascript", "/_pdf_js/pdf.js"),
                 (os.path.join(here, "..", "..", "third_party", "pdf_js", "pdf.worker.js"), None,
                  "text/javascript", "/_pdf_js/pdf.worker.js"),
-                (self.options.get("testharnessreport", "testharnessreport.js"),
+                (self.options.get("testharnessreport", [os.path.join("executors", "message-queue.js"),
+                                                        "testharnessreport.js"]),
                  {"output": self.pause_after_test,
                   "timeout_multiplier": self.testharness_timeout_multipler,
                   "explicit_timeout": "true" if self.debug_info is not None else "false",
                   "debug": "true" if self.debug_test else "false"},
                  "text/javascript;charset=utf8",
-                 "/resources/testharnessreport.js")]:
-            path = os.path.normpath(os.path.join(here, path))
+                 "/resources/testharnessreport.js"),
+                ([os.path.join(repo_root, "resources", "testdriver.js"),
+                  os.path.join("executors", "message-queue.js"),
+                  "testdriver-extra.js"], {}, "text/javascript", "/resources/testdriver.js")]:
```

Comment:
Can we format L233 - L243 a little bit better?

Reply:
Use 'git cl format' in chromium, I got something like below:

+        [(self._options.get("testharnessreport", [
+            os.path.join("executors", "message-queue.js"),
+            "testharnessreport.js"
+        ]), {
+            "output": self.pause_after_test,
+            "timeout_multiplier": self.testharness_timeout_multipler,
+            "explicit_timeout":
+            "true" if self.debug_info is not None else "false",
+            "debug": "true" if self.debug_test else "false"
+        }, "text/javascript;charset=utf8", "/resources/testharnessreport.js"),
+         ([
+             os.path.join(repo_root, "resources", "testdriver.js"),
+             os.path.join("executors", "message-queue.js"),
+             "testdriver-extra.js"
+         ], {}, "text/javascript", "/resources/testdriver.js")]